spec/features/dams_collections_spec.rb

```diff
@@ -81,7 +81,40 @@
     visit dams_collections_path({:per_page=>100})
     expect(page).to have_content('Access: Curator Only')
   end
+end
+
+feature 'Visitor wants to see the search result page default sort is title after click "View Collection Items" url' do
+  before do
+    @unit = DamsUnit.create(pid: 'bb45454545')
+    @unit.damsMetadata.content = File.new('spec/fixtures/damsUnit.rdf.xml').read
+    @unit.save!  
+    @aCollection = DamsAssembledCollection.create(pid: "xx4473712z", visibility: "public", titleValue: "testCollection" )
+    @aCollection.save!
+    @testObj1 = DamsObject.create(pid: "xx2801340v")
+    @testObj1.damsMetadata.content = File.new('spec/fixtures/soccomObject1.rdf.xml').read
+    @testObj1.save!
+    @testObj2 = DamsObject.create(pid: "xx47126209")
+    @testObj2.damsMetadata.content = File.new('spec/fixtures/soccomObject2.rdf.xml').read
+    @testObj2.save!
+    solr_index (@unit.pid)
+    solr_index (@aCollection.pid)
+    solr_index (@testObj1.pid)
+    solr_index (@testObj2.pid)
+  end
+  after do
+    @aCollection.delete
+    @unit.delete
+    @testObj1.delete
+    @testObj2.delete
+  end 
 
+  scenario 'The search result page default sort should be title' do
+    visit dams_collection_path("#{@aCollection.pid}")
+    first(:link, "View Collection Items").click
+    
+    expect(page).to have_content("Sort: title")
+    expect(page).to_not have_content("Sort: relevance")
```

Comment:
@mcritchlow The functionality of sort by title (by relevance and etc. ) is the blacklight built-in function. Here we want to make sure when load the page the result page has the "Sort: title" as default .
If sort: title displayed but the page is not actually sort by title, it will be blacklight built-in function bug. So Would it be redundant to write those tests again in our local app?

Reply:
Yeah, that's a very fair point. I suppose the main (only?) value-add that I can think of for adding an `expect` saying "I also expect this object to be shown before that object in the list" just makes it explicit that this is why we're making this change. If it wasn't a feature, or end-to-end test, I'd probably feel different.

But that said, your point is well taken, and I don't have that strong feelings one way or the other. We could certainly leave it as-is.